### PR TITLE
gherkin_script: Display failure details on pipeline run details

### PR DESCRIPTION
**Pull request by sanketpathak**
https://issues.redhat.com/browse/ODC-4257

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/pipelines/pipeline-Display failure details on pipelineruns and taskruns-ODC-3713.feature [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Feature: Add page for Pipeline
+	As a user, I want see failure details on pipelineruns and taskruns
+
+Background:
+    Given Pipelines operator is installed
```
[the remark this answers is not in the document]
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/pipelines/pipeline-Display failure details on pipelineruns and taskruns-ODC-3713.feature]: We have used Pipeline operator term in other files as well so its appropriate to use it although I've done the required changes but it is not required

File: frontend/packages/dev-console/integration-tests/features/pipelines/pipeline-Display failure details on pipelineruns and taskruns-ODC-3713.feature [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+   When user goes to failed pipeline run
+   And user opens pipeline run details
+   Then user can see status as Failure
+   And user can view failure message under Message heading
+   And user can see Log snippet to get know what taskruns failed
+
+@regression
+Scenario: Display failure details of pipeline run in topology sidebar
+   Given user is in topology
+   And a node with failed pipeline is present
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/integration-tests/features/pipelines/pipeline-Display failure details on pipelineruns and taskruns-ODC-3713.feature:20]: Not sure how much assumption here is... but did we want to say what "node with failed pipeline" is? Like
```suggestion
   And a node with an associated pipeline that has failed is present
```

File: frontend/packages/dev-console/integration-tests/features/pipelines/pipeline-Display failure details on pipelineruns and taskruns-ODC-3713.feature [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+   When user opens sidebar of the n ode
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/integration-tests/features/pipelines/pipeline-Display failure details on pipelineruns and taskruns-ODC-3713.feature:21]:
```suggestion
   When user opens sidebar of the node 
```

File: frontend/packages/dev-console/integration-tests/features/pipelines/pipeline-Display failure details on pipelineruns and taskruns-ODC-3713.feature [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+@regression
+Scenario: Display failure details of pipeline run in topology sidebar
+   Given user is in topology
+   And a node with failed pipeline is present
+   When user opens sidebar of the n ode 
+   And user scrolls down to pipeline runs section
+   Then user will see the pipeline run name with failed status 
+   And user will see failure message below pipeline runs
+   And user will also see the log snippet with horizontal scroller
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/integration-tests/features/pipelines/pipeline-Display failure details on pipelineruns and taskruns-ODC-3713.feature:25]: Technically there is nothing saying it _has_ to have a horizontal scroller. It is technically possible not to have it.

**Pull request merged**